webapp/src/Controller/Jury/ConfigController.php

```diff
@@ -85,8 +85,26 @@ public function indexAction(EventLogService $eventLogService, Request $request):
             }
 
             if (empty($errors)) {
-                $this->addFlash('scoreboard_refresh', 'After changing specific ' .
-                    'settings, you might need to refresh the scoreboard.');
+                $needsRefresh = false;
+                $needsRejudging = false;
+                foreach ($diffs as $key => $diff) {
+                    $category = $this->config->getCategory($key);
+                    if ($category === 'Scoring') {
+                        $needsRefresh = true;
+                    }
```

Comment:
Can't you add an break here as you only need 1?

Reply:
No, there is two separate booleans here.